### ISSUE TITLE
Xkcd: Add explain xkcd link

### ISSUE THREAD

**Issue by swapagarwal** [swapagarwal/JARVIS-on-Messenger#130]
e.g. http://xkcd.com/1767/ and http://explainxkcd.com/1767/
Add links using ButtonTemplate.

**Comment by CassiusYCai** [comment]
Hi @swapagarwal I'd love to work on this!

**Comment by CassiusYCai** [comment]
Hi @swapagarwal, I've been able to create the Button to add the explainxkcd link, but I'm receiving the error message whenever I input "xkcd" as my query, so I am unable to test it (even without my changes). 

I believe I've narrowed the error to line 10 in xkcd.py, specifically the request "with requests_cache.enabled..." but do not understand what is happening on that line. I've looked online for documentation, but am having a hard time understanding it. Any chance you could point me to a better source for where to look for more understandable documentation so I can make sense of what is trying to be done?

**Comment by swapagarwal** [comment]
@CassiusYCai Cool, you can go ahead and create a Pull Request, and we can take it from there! 😄 
Can you tell what error message are you getting?
The `requests_cache` caches the request on the server so that we do not have to hit the same URL again and again!
On a high level, you'd have to add two buttons, like the [`music`](https://github.com/swapagarwal/JARVIS-on-Messenger/blob/master/modules/src/music.py) module.

**Comment by CassiusYCai** [comment]
Hey @swapagarwal Thanks so much for the reply and the help. Just finished taking a look at the music module and used the framework to create two buttons in the xkcd file.

Going back to basics, I'm having trouble creating a PR: I created a separate branch, added and committed my changes, but am getting the following message when I try to push: 

> remote: Permission to swapagarwal/JARVIS-on-Messenger.git denied to CassiusYCai.
> fatal: unable to access 'https://github.com/swapagarwal/JARVIS-on-Messenger.git/': The requested URL returned error: 403

After some googling, I created another private key, and added it to my account but am still coming up short. Is there something I am doing wrong/what other avenue should I take to create the PR?

Thanks again for the help! You're the best :)
